setup.py

```diff
@@ -55,9 +55,21 @@ def get_version():
         for line in lines:
             if line.startswith('Version:'):
                 return line[8:].strip()
-    if os.path.exists("./tools"):
-        print("Running tools/get_version.sh")
-        version = subprocess.getoutput("bash -c 'cd tools; ./get_version.sh'")
+    if os.path.exists("./include/vrv"):
+        with open("./include/vrv/vrvdef.h") as header_file:
+            defines = {}
+            for line in header_file.readlines():
```

Comment:
You don't actually need "`readlines()`" -- `for line in header_file` should work.

Reply:
Done.